lib/sorcery/model/temporary_token.rb

```diff
@@ -12,7 +12,7 @@ def self.included(base)
 
       # Random code, used for salt and temporary tokens.
       def self.generate_random_token
-        SecureRandom.urlsafe_base64(15).tr('lIO0', 'sxyz')
+        SecureRandom.urlsafe_base64(Controller::Config.token_randomness).tr('lIO0', 'sxyz')
```

Comment:
i think this is not safety enough.
can you consider to do something like below?

```ruby
SecureRandom.urlsafe_base64([15, Controller::Config.token_randomness].max).tr('lIO0', 'sxyz')
```